util/deriveExp2QuatDerivatives.m

```diff
@@ -0,0 +1,55 @@
+v = sym('v',[3,1]);
```

Comment:
actually, perhaps this should move to dev?  you don't really mean for anybody except us to call it... (including the build servers), right?